components/FormInput.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { Button, Text, TextInput, View } from "react-native";
+
+const FormText = styled.TextInput`
+  margin-bottom: 10;
+  border-bottom-width: 2;
+  border-bottom-color: black;
+`;
+
+const FormInput = ({ submitHandler }) => {
+  const [text, setText] = useState("");
+
+  const changeHandler = (value) => {
+    setText(value);
+  };
+
+  return (
+    <View>
+      <FormText
+        placeholder="New todo..."
+        onChangeText={(value) => changeHandler(value)}
+        value={text}
+      />
+      <Button
+        onPress={() => {
+          submitHandler(text);
+          setText("");
```

Comment:
If we need to pass more than one line of code to event handler, then it's clear indicator for us to create separate function